app/javascript/optimization/listen.js

```diff
@@ -0,0 +1,15 @@
+import { refresh } from './refresh.js';
+
+export function miqOptimizationInit() {
+  window.listenToRx((data) => {
+    if (data.controller != 'OptimizationController') {
```

Comment:
i think linter will be complaining about `!=` instead of `!==`

Reply:
fixed